api/agent_client.go

```diff
@@ -23,6 +23,10 @@ func NewAgentClient(token, endpoint, clusterID, queue string, agentQueryRules []
 	if err != nil {
 		return nil, err
 	}
+	if queue == "" {
+		// This is a special keyword on backend.
+		queue = "_default"
```

Comment:
perhaps it's `const`ifying the `"_default"` string, if only just to indicate that it's Important

Reply:
@sj26 it's implemented in https://github.com/buildkite/buildkite/pull/23261 

@moskyb yep I constified this string.

Reply:
Interesting!

Why a special value, why not treat empty/missing as "default"?

Reply:
@sj26 I would've asked the same question. My first thought was to use empty to represent "default" but it has been tricky given the url structure is: `/cluster/~/queue/~/scheduled_jobs`, allowing empty queue bit will confuse the router. 

My second thought was to pull default queue key before making this request, but it will suffer issue when default queue changes on backend. 

So, with the understanding that empty value is a subset of "special values", we end up using special keyword. 

The special keyword is safe because a real queue key has to start with alphanumeric characters.